config/plugin/network_problem.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This plugin checks for common network issues. Currently, it only checks
+# if the conntrack table is full. 
+
+conntrack_max=$(cat /proc/sys/net/ipv4/netfilter/ip_conntrack_max)
```

Comment:
I think we need to check the file existence firstly, otherwise if one of the two files do not exit, the check script will return error exit code.

Reply:
Done